### PR TITLE
setup minikube before env setup

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this?**
/kind tests

**What does this PR do / why we need it**:
This PR fixes the failing k8s test. 

**Which issue(s) this PR fixes**:

Fixes #?

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [x] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by anandrkskd** [comment]
failure in debug test
/retest

File: scripts/minikube-minishift-setup-env.sh [hunk 1]
```diff
@@ -6,51 +6,48 @@ shout() {
...
-# Add GOBIN which is the bin dir we created earlier to PATH so any binaries there are automatically available in PATH
-export PATH=$PATH:$GOBIN
+setup_operator() {
+  SETUP_OPERATORS="./scripts/configure-cluster/common/setup-operators.sh"
 
-# Prep for integration/e2e
-shout "Building odo binaries"
-make bin
+  # The OLM Version
+  export OLM_VERSION="v0.17.0"
```
Comment (dharmit) [inline scripts/minikube-minishift-setup-env.sh:16]: We can (should?) use newer version now. 0.19.1 is available. We had earlier kept this at 0.17.0 due to issues with 0.18.2.

Comment (dharmit) [reply scripts/minikube-minishift-setup-env.sh:16]: Another way to not hard code this in script and pull artefacts from latest could be to do:
```sh
LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/operator-framework/operator-lifecycle-manager/releases/latest)
LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
```
Now `echo $LATEST_VERSION` will have the value. I found this from https://blog.markvincze.com/download-artifacts-from-a-latest-github-release-in-sh-and-powershell/.

File: scripts/minikube-minishift-setup-env.sh [hunk 2]
```diff
@@ -6,51 +6,48 @@ shout() {
...
+  # Enable OLM for running operator tests
+  curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/$OLM_VERSION/install.sh | bash -s $OLM_VERSION
 
-# copy built odo to GOBIN
-cp -avrf ./odo $GOBIN/
+  # Create Operators for Operator tests
```
Comment (dharmit) [inline scripts/minikube-minishift-setup-env.sh:20]:
```suggestion
  # Install Operators for Operator tests
```
Being pedantic here, but we aren't creating any Operators, we are just using them. 😉

**Review by dharmit — CHANGES_REQUESTED** [review]

File: scripts/minikube-minishift-setup-env.sh (hunk 1 above)
Comment (anandrkskd) [reply scripts/minikube-minishift-setup-env.sh:16]: Understood! I will update this accordingly
Thanks

File: scripts/minikube-minishift-setup-env.sh (hunk 2 above)
Comment (anandrkskd) [reply scripts/minikube-minishift-setup-env.sh:20]: 😅 will update this too

**Comment by feloy** [comment]
/test psi-kubernetes-integration-e2e

```
 [Fail] odo link command tests for OperatorHub Operators are installed in the cluster when a component and a service are deployed when a link with between the component and the service is created with --bind-as-files and --inlined when odo push is executed [BeforeEach] should display the link in odo describe 
```

**Comment by anandrkskd** [comment]
@feloy, @dharmit, can you lgtm and approve this PR.

**Pull request merged**